ecl2df/wellcompletiondata.py

```diff
@@ -17,6 +18,13 @@
 logger = logging.getLogger(__name__)
 
 
+class EclipseUnitSystem(str, Enum):
```

Comment:
Do you gain something from subclassing form `str` also?

Reply:
Yes, because I do this in line 100:

`            return EclipseUnitSystem(keyword.name)`